sqlmesh/dbt/basemodel.py

```diff
@@ -55,6 +57,8 @@ class Hook(DbtConfig):
     sql: SqlStr
     transaction: bool = True  # TODO not yet supported
 
+    _sql_validator = sql_str_validator
```

Comment:
Is this used? I didn't see it called in the diff.

Reply:
This is a pydantic validator. What do you mean by "called"? Pydantic calls this

Reply:
For context: Pydantic V2 no longer automatically coerces regular strings into `SqlStr`, so a validator is required.